speech/api/speech_async_grpc.py

```diff
@@ -59,8 +59,12 @@ def main(input_uri, encoding, sample_rate):
     # https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1beta1/cloud_speech.proto
     response = service.AsyncRecognize(cloud_speech_pb2.AsyncRecognizeRequest(
         config=cloud_speech_pb2.RecognitionConfig(
-            encoding=encoding,
-            sample_rate=sample_rate,
+            # There are a bunch of config options you can specify. See
+            # https://g.co/cloud/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig  # noqa
+            # for the full list.
+            encoding=encoding,  # one of LINEAR16, FLAC, MULAW, AMR, AMR_WB
+            sample_rate=sample_rate,  # the rate in Hz
```

Comment:
Hrm... is it capitalized when spelled out? Makes it look like the car company...